migrations/versions/e46b30dd3ff5_automatic_collections.py

```diff
@@ -0,0 +1,76 @@
+"""automatic_collections
+
+Revision ID: e46b30dd3ff5
+Revises: 9e33fa16ba82
+Create Date: 2023-02-06 11:44:29.000880
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e46b30dd3ff5'
+down_revision = '9e33fa16ba82'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "automatic_collection",
+        sa.Column(
+            'collection_id',
+            sa.Integer,
+            sa.ForeignKey('collections.id', ondelete='CASCADE'),
+            primary_key=True,
+        ),
+        sa.Column(
+            'keyword_query',
+            sa.String,
+        ),
+        sa.Column(
+            'author_query',
+            sa.String,
+        ),
+        sa.Column(
+            'title_query',
+            sa.String,
+        ),
+        sa.Column(
+            'subject_query',
+            sa.String,
+        ),
+        sa.Column(
+            'sort_field',
+            sa.String,
+            sa.CheckConstraint(r"sort_field is NULL OR sort_field IN ('title', 'author', 'date')"),
+        ),
+        sa.Column(
+            'sort_direction',
+            sa.String,
+            sa.CheckConstraint(r"sort_direction is NULL OR sort_direction IN ('ASC', 'DESC')"),
+        ),
+        sa.Column(
+            'limit',
+            sa.Integer,
+            nullable=False,
+        ),
```

Comment:
Should `limit` be required? I see the need for some kind of functionality here but what if we wanted to create a collection around an author and as our catalog grows it exceeds this limit? I could see an argument for a max collection size being defined somewhere universally, but this might create some unexpected behavior?

Reply:
hmm good point, perhaps we could require *either* one of the `_query` fields *or* (inclusive) the `limit` field. 

Though now I wonder if there's any utility to having a limit at all if I add a bit of logic to https://github.com/NYPL/drb-etl-pipeline/pull/225 to implement pagination in the DB query (and elastic query for search based stuff).  I guess a question for Sean that I can reach out about.

Reply:
I think I'll just make `limit` optional and do the pagination thing I mentioned in the next PR.